### PR TITLE
chore: use git town for making stacked changes

### PR DESCRIPTION
**Pull request by RogerHYang**
https://www.git-town.com/questions

### CODE REVIEW

File: .git-branches.toml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Git Town configuration file
```
Comment (mikeldking) [inline .git-branches.toml:1]: is this default? I don't really know what all these things actually do 😓

File: .github/workflows/git-town.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Git Town
```
Comment (mikeldking) [inline .github/workflows/git-town.yml:1]: Can you add more color to the name? I'm not sure contributors will know what git town is

Comment (mikeldking) [reply .github/workflows/git-town.yml:1]: seems to be failing?

Comment (RogerHYang) [reply .github/workflows/git-town.yml:1]: Yea, i'm not really sure what's going on: Google search didn't find anything. Maybe it'll work in real life? If not, we can always revert it

File: .git-branches.toml (hunk 1 above)
Comment (RogerHYang) [reply .git-branches.toml:1]: yea, this pretty much the default. We can always change it, they're innocuous.

**Review by mikeldking — APPROVED** [review]

**Pull request merged**